docker/sorry-cypress/sorry-cypress.yml

```diff
@@ -0,0 +1,112 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Sorry Cypress Setup
+# ----------------------------------------------------------------------------------------------------------------------
+
+services:
+  mongo:
+    image: mongo:4.4
```

Comment:
In the other Artemis setups we always use the more specific docker registry URL like in this case:
```suggestion
    image: docker.io/library/mongo:4.4
```

Reply:
Done